docs/upgrade-guide/upgrade211-30.md

```diff
@@ -0,0 +1,142 @@
+Upgrade guide ACA V3.0 application compitable with ADF V5.0
+
+---
+
+Title: Upgrading from ACA v2.11 to v3.0
+
+---
+
+# Upgrading from ACA v2.11 to v3.0
```

Comment:
this guide does not describe how to upgrade to the ADF, only contains pieces of the Angular upgrade guide... given that ADF has breaking changes, what should be done in the application to compile?

Reply:
I have checked & verified there is angular upgrade to V14 in ADF 5.0. So, I have written separate upgrade guide in ADF repo.

Reply:
this guide must help people upgrade everything, it's not the Angular upgrade guide, but the ACA upgrade to ADF having Angular alongside... have you ever seen upgrade guides before?

Reply:
Yes, you are right. I got your point. I am updating it.

Reply:
@rbahirsheth upgrade guide for ADF is totally different thing, in this guide you need to provide instructions on what users should do to make ACA compatible with upgraded ADF and how to fix breaking changes that will appear